python_modules/dagster/dagster_tests/core_tests/structured_config_tests/test_structured_config_types.py

```diff
@@ -439,3 +448,79 @@ def a_job():
         }
     )
     assert executed["yes"]
+
```

Comment:
would be nice to see the non-noneable case with a `None` value provided covered under test.

Reply:
:hmmnod: